lib/static.js

```diff
@@ -20,6 +22,12 @@ module.exports = function renderStaticMarkup(options) {
   awaits.push(markupAwait);
   awaits.push(highlightAwait);
 
+  if(!_.isEmpty(options.scripts)) {
+    const scriptsDir = path.join(targetPath, 'scripts');
+    shell.mkdir('-p', scriptsDir);
+    shell.cp(options.scriptSources, scriptsDir);
```

Comment:
Thanks, @WJWang. Could you please use the dependency [fs-extra](https://github.com/jprichardson/node-fs-extra) for this?